### PR TITLE
[US749252] - Flags to disable ssl pinning.

### PR DESCRIPTION
**Pull request by ysd24**
[US749252] - Flags to disable ssl pinning.

### CODE REVIEW

**Review by abhisheknaidu-462 — APPROVED** [review]
LGTM

File: MASFoundation/Classes/MAS.h [hunk 1]
```diff
@@ -157,6 +157,26 @@
...
+*
+*  @param enable BOOL value of indicating whether SSL Pinning should be enabled or not.
+*/
++ (void)enableSSLPinning:(BOOL)enable;
+
+
+
+/**
+*  Gets BOOL indicator of SSL Pinning enabled or not for Server Trust evluation of pinned Server Certificates.
+*  By default, SSL Pinning is disabled.
```
Comment (zakirjt) [inline MASFoundation/Classes/MAS.h]: By default, ssl pinning is enabled right ?, same thing on line 162 also.

Comment (ysd24) [reply MASFoundation/Classes/MAS.h]: Done.

**Review by zakirjt — APPROVED** [review]

**Review by arindamhit — APPROVED** [review]
Looks good

**Pull request merged**